server.js

```diff
@@ -1,20 +1,18 @@
-'use strict';
+"use strict";
 
-const express     = require('express');
-const bodyParser  = require('body-parser');
-const fccTesting  = require('./freeCodeCamp/fcctesting.js');
+const express = require("express");
+const fccTesting = require("./freeCodeCamp/fcctesting.js");
 
 const app = express();
 
 fccTesting(app); //For FCC testing purposes
-app.use('/public', express.static(process.cwd() + '/public'));
-app.use(bodyParser.json());
-app.use(bodyParser.urlencoded({ extended: true }));
+app.use("/public", express.static(process.cwd() + "/public"));
+app.use(express.json());
+app.use(express.urlencoded({ extended: true }));
 
-app.route('/')
-  .get((req, res) => {
-    res.sendFile(process.cwd() + '/views/index.html');
-  });
+app.route("/").get((req, res) => {
+  //Write your code here to render the Pug template
```

Comment:
```suggestion
  //Write your code here to render the Pug template
  res.send('pug template should be rendered instead of this');
```
I still wonder if this would be better than having the site hang until the user fixes it, but I'll defer to @scissorsneedfoodtoo on this one.

Reply:
ok. i added that response.  Makes sense to show something in the response. I changed to master. If you are going to merge, make sure you set it as default.  I don't have that access.